MFE/container/src/i18n/translations/pt_br.success.i18n.json

```diff
@@ -24,7 +24,7 @@
     "deleteMultipleCertificates": "Todos os certificados foram excluídos",
     "disassociateDevice": "Certificado desassociado do dispositivo",
     "associateDevice": "Certificado associado ao dispositivo",
-    "createCertificate": "Certificado criado",
```

Comment:
Here, I have removed this line because there is already a translation for 'createDevice' on line 22 of this file